server/src/test/java/org/opensearch/search/query/QueryProfilePhaseTests.java

```diff
@@ -477,6 +485,14 @@ public void testTerminateAfterEarlyTermination() throws Exception {
                 assertThat(query.getTimeBreakdown().keySet(), not(empty()));
                 assertThat(query.getTimeBreakdown().get("score"), greaterThan(0L));
                 assertThat(query.getTimeBreakdown().get("score_count"), greaterThan(0L));
+                if (query.getMaxSliceTime() != null) {
```

Comment:
Same comment as above tests to use concurrency count here to differentiate between concurrent vs non concurrent case